src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -307,7 +307,7 @@ const ChartsHolder = (props: {
                   <Outcomes
                     title={`Predicted outcomes by ${formatUtcDate(
                       props.projections.projected.finalDate,
-                    )} (90 days from now)`}
+                    )} (30 days from now)`}
```

Comment:
nit: since you exported the PROJECTIONS_TRUNCATION_DAYS constant, you could use that here.